### DISCUSSION TITLE
Microsoft.FluentUI.AspNetCore.Components.Color to Hex?

### DISCUSSION THREAD

**Discussion by SpenceDahl** [microsoft/fluentui-blazor#2762] [Q&A] [answered]
How do I use Microsoft.FluentUI.AspNetCore.Components.Color[s] in a component like `<FluentBadge Color="hexValue">` ? I'm looking for something like:        

 `Color myFluentColor = Color.Accent; // Using a predefined Fluent UI color`

`string hexValue = myFluentColor.ToHex(); // Assuming a method to convert to hex `
 
 I need a ToHex() method (or what's a better way).

**Comment by vnbaaij** [comment, marked as answer]
The whole idea is that you do not have to deal with the hex value. The accent color is set to a value through the script and the AccentFillRest design token which then translates to setting the CSS var --accent-fill-rest. 
You can get the associated CSS var name by using out `ToAttibuteValue()` enum extension method.
